docs/explanations/faq.rst

```diff
@@ -1,23 +1,68 @@
+  4. If the model still fails to solve, try using the Degeneracy Hunter or SVD toolboxes to look for more complex issues.
+  5. In some cases, it can help to try a different solver (if you have access to one). Different solvers use different approaches to solving the problem, and sometimes one will succeed where others fail (especially on mildly ill posed problems).
+  6. If all else fails, see the “How do I get more help?” section below. When reaching out for help, please include the full output of the IDAES and solver logs, as well as any issues identified by the diagnostics toolbox.
+
+
+Help! My model raises Exceptions during construction!
+`````````````````````````````````````````````````````
+
+Before contacting the IDAES for assistance or filing a bug report, please do the following checks:
 
-How do I...
------------
+  1. First, check the error message you received and try to understand what it is telling you and where it is coming from (IDAES, Pyomo or Python). Often, the trick to understanding error message is not understanding what went wrong, but how it happened. If the error appears to be coming from Python or Pyomo, then it is most likely a bug somewhere in the code.
+  2. Second, check the full Python traceback of the error to try to identify where in the code it is originating. This will help narrow down the possible issues and allow you to isolate the offending code for testing. It is always easier to debug a smaller model than a large flowsheet (especially for an IDAES developer who is not familiar with the flowsheet you are trying to solve).
+  3. If you cannot find the source of the error yourself, try to make a minimum failing example which replicates the problem that you can share with the IDAES team, as this will great increase the chances of us having the time to test your example and isolate the issue.
+  4. To get help, see the “How do I get more help?” section below. When reaching out for help, please include the full Python traceback and any warnings from the logger, as well as the minimum failing example.
 
-... Run some examples?
-    See :doc:`../tutorials/tutorials_examples`.
 
-... Get more help?
-    Use the website to `register <https://idaes.org/register/>`_ for the IDAES support mailing list.
-    Then you can send questions to idaes-support@idaes.org. For more specific technical questions, we recommend
-    our new `IDAES discussions board on Github <https://github.com/IDAES/idaes-pse/discussions>`_.
+How do I run some examples?
+```````````````````````````
+
+See :doc:`../tutorials/tutorials_examples`.
+
+How do I get more help?
+```````````````````````
+
+Use the website to `register <https://idaes.org/register/>`_ for the IDAES support mailing list.
+Then you can send questions to idaes-support@idaes.org. For more specific technical questions, we recommend
+our new `IDAES discussions board on Github <https://github.com/IDAES/idaes-pse/discussions>`_.
```

Comment:
I would not say "new" as that goes stale quickly.  Maybe `we also recommend the IDAES discussions board on GitHub`